.gitignore

```diff
@@ -0,0 +1,96 @@
+##
+# .gitignore
+#
+# Based on:
+#
+# https://github.com/github/gitignore/blob/master/Global/OSX.gitignore
+# https://github.com/github/gitignore/blob/master/Python.gitignore
+##
+
+.DS_Store
+.AppleDouble
+.LSOverride
```

Comment:
Personally I think these sorts of things ought to be in people's global `core.excludesfile`, but since we're in a Mac monoculture I might be persuaded 😉

Reply:
The duplication doesn't really hurt anything anyways